Source/Runtime/MeasurementLink Discovery V1/MeasurementLink Discovery V1.lvlib

```diff
@@ -29,7 +29,7 @@
 	</Item>
 	<Item Name="Discovery Service Helpers" Type="Folder">
 		<Item Name="Ensure Discovery Service Started.vi" Type="VI" URL="../Ensure Discovery Service Started.vi">
-			<Property Name="NI.LibItem.Scope" Type="Int">2</Property>
+			<Property Name="NI.LibItem.Scope" Type="Int">4</Property>
```

Comment:
This doesn't appear to be called from outside the library. I think it should remain private. Also, `Get Discovery Service Key File Path.vi` also doesn't appear to be called from outside the library. I would make it private as well.

Reply:
Changed the access of mentioned VIs to private